project/Dependencies.scala

```diff
@@ -16,19 +16,19 @@ object Dependencies {
 
   val Smithy = new {
     val org = "software.amazon.smithy"
-    val version = "1.26.4"
-    val model = org % "smithy-model" % version
-    val testTraits = org % "smithy-protocol-test-traits" % version
-    val build = org % "smithy-build" % version
-    val awsTraits = org % "smithy-aws-traits" % version
-    val waiters = org % "smithy-waiters" % version
+    val smithyVersion = "1.26.4"
```

Comment:
**suggestion:** shall we bump to 1.27.0 now that we're at it?

Reply:
I figured that'd be a good test to see whether or not the bot actually sees a difference, wdyt?

Reply:
I did alloy, because I sort of need it hehe